operators/src/error.rs

```diff
@@ -314,6 +314,20 @@ pub enum Error {
     QueryCanceled,
 
     AbortTriggerAlreadyUsed,
+
+    SubPathMustNotEscapeBasePath {
+        base: PathBuf,
+        sub_path: PathBuf,
+    },
+
+    InvalidDataProviderConfig,
+    MachineLearningModelNotAvailable,
```

Comment:
better call in ...NotAvailable

Reply:
Changed with 99d0b9ff66a24c4e316ac5f3e0c44e051d4cb69f